### PR TITLE
Update current network info (chain, genesis hash) when connected

### PR DESCRIPTION
**Pull request by ashchan**
Whenever connecting to a new network or current one, it should update genesis info and chain type.

### CODE REVIEW

File: packages/neuron-wallet/src/main.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { changeLanguage } from 'utils/i18n'
 import env from 'env'
 import { register as registerListeners } from 'listeners/main'
 
-const appController = new AppController()
+let appController = new AppController()
```
Comment (classicalliu) [inline packages/neuron-wallet/src/main.ts]: What's the purpose here ?

File: packages/neuron-wallet/src/services/networks.ts [hunk 2]
```diff
@@ -100,24 +91,7 @@ export default class NetworksService extends Store {
...
-      }
-
-      const genesisHash = await core.rpc
-        .getBlockHash('0x0')
-        .catch(() => EMPTY_GENESIS_HASH)
-      if (genesisHash !== EMPTY_GENESIS_HASH) {
-        network.genesisHash = genesisHash
-      }
-    }
+    Object.assign(await this.refreshChainInfo(network))
```
Comment (classicalliu) [inline packages/neuron-wallet/src/services/networks.ts]: Only one argument here ?

Comment (ashchan) [reply packages/neuron-wallet/src/services/networks.ts]: Oops

File: packages/neuron-wallet/src/main.ts (hunk 1 above)
Comment (ashchan) [reply packages/neuron-wallet/src/main.ts]: fixed.

File: packages/neuron-wallet/src/services/networks.ts (hunk 2 above)
Comment (ashchan) [reply packages/neuron-wallet/src/services/networks.ts]: Updated.

**Pull request merged**